### ISSUE TITLE
Fail to parse base64 float value in put command

### ISSUE THREAD

**Issue by weichou1229** [edgexfoundry/device-sdk-go#453]
For example,
I can execute read command:
```
curl http://localhost:48082/api/v1/device/name/Test-Device/command/GenerateDeviceValue_FLOAT64_RW 
{
   "origin" : 1582262953501573300,
   "EncodedEvent" : null,
   "readings" : [
      {
         "name" : "DeviceValue_FLOAT64_RW",
         "value" : "P/rJxH30EAY=",
         "device" : "Test-Device",
         "origin" : 1582262953496863800
      }
   ],
   "device" : "Test-Device"
}
```

When I try the write command with the same base64 string **P/rJxH30EAY=**
```
curl http://localhost:48082/api/v1/device/name/Test-Device/command/GenerateDeviceValue_FLOAT64_RW  \
  -X PUT -H 'Content-Type: application/json' \
  -d '{"DeviceValue_FLOAT64_RW":"P/rJxH30EAY="}'

Handler - execWriteCmd: Put parameters parsing failed: {"DeviceValue_FLOAT64_RW":"P/rJxH30EAY="} /api/v1/device/a93d22b2-dc64-4d39-8506-b995965521b0/GenerateDeviceValue_FLOAT64_RW
```
Error logs:
```
level=ERROR ts=2020-02-21T05:30:44.3333032Z app=device-virtual source=command.go:495 msg="Handler - Command: Parsing parameter value (P/rJxH30EAY=) to Float64 failed: strconv.ParseFloat: parsing \"P/rJxH30EAY=\": invalid syntax"
level=ERROR ts=2020-02-21T05:30:44.3334222Z app=device-virtual source=command.go:314 msg="Handler - execWriteCmd: Put parameters parsing failed: {\"DeviceValue_FLOAT64_RW\":\"P/rJxH30EAY=\"}"
```

It seems SDK didn't handle the base64 for parsing the float value.
https://github.com/edgexfoundry/device-sdk-go/blob/master/internal/handler/command.go#L481

**Comment by hahattan** [comment]
is this issue resolved in #499 ? @cloudxxx8

**Comment by cloudxxx8** [comment]
yes, fixed by #499
